tests/zepkit/test/src/integration.test.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+require('chai').should();
+const _ = require('lodash'); 
+
+const { 
+  getNetwork,
+  run,
+} = require('../../../util/share')(__dirname);
+
+function cleanup() {
+  // haha, famous line
+  run('cd .. && rm -rf workdir && mkdir workdir');
+}
+
+const network = getNetwork();
+
+function runIntegrationTest({ kit }, action) {
+  before('cleaning up project folder', cleanup);
+
+  it('unpack kit', function () {
+    run(`npx zos unpack ${kit}`);
```

Comment:
Big question here: what `zos` is being used here? We need to make sure the current version from CLI is being used, and *not* the one published to npm.

Reply:
You got me on this one 😅  NPX would pull down latest version from npm. Let me figure it out. There is even a bigger problem that Hot Loader will pull either local or global version. It would be perfect to trick npx to use dev version. I wonder is it possible?

Reply:
It should be. If there is a `package.json` on the workdir, that points to the same `zos` version as in the current project, then `lerna bootstrap` **should** turn that dependency into a symlink to the local package folder.

At least, that's how the current `test/cli` is supposed to be working.

(as a side note, I'm considering removing the need for lerna bootstrap and replace it with just file references, which seems to be the latest recommended way of using lerna; but let's stick with this method for now)